uni/assets/text/TermsAndConditions.md

```diff
@@ -1,138 +1,138 @@
-right to use of all submitted content.
+right to use of all submitted Content.
 \
 \
-**data usage, monitoring and privacy**
+**Data usage, Monitoring and Privacy**
 
-we may, at our discretion but with no obligation, monitor and review content in the mobile
-application submitted or created using our services by you.
-personal data of users is not collected for any other purpose, unless providing the services to you,
+We may, at our discretion but with no obligation, monitor and review Content in the Mobile
+Application submitted or created using our Services by you.
+Personal data of users is not collected for any other purpose, unless providing the Services to you,
 except in certain circumstances described
-in [play store’ privacy policy](https://support.google.com/googleplay/android-developer/answer/10144311?visit_id=638365485539535125-3072678242&rd=1)
-and [app store's privacy policy](https://developer.apple.com/app-store/app-privacy-details/#user-tracking).
+in [Play Store’ Privacy Policy](https://support.google.com/googleplay/android-developer/answer/10144311?visit_id=638365485539535125-3072678242&rd=1)
+and [App Store's Privacy Policy](https://developer.apple.com/app-store/app-privacy-details/#user-tracking).
 \
 \
-**license and usage restrictions:**
+**License and Usage Restrictions:**
 
-you grant us permission to access, copy, distribute, store, transmit, reformat, display and perform
-the content of your user account solely as required for the purpose of providing the services to
+You grant us permission to access, copy, distribute, store, transmit, reformat, display and perform
+the Content of your user account solely as required for the purpose of providing the Services to
 you.
-unless specifically permitted by you, your use of the mobile application does not grant us the
-license to link, reproduce, adapt, modify, publish or distribute the content created by you or
+Unless specifically permitted by you, your use of the Mobile Application does not grant us the
+license to link, reproduce, adapt, modify, publish or distribute the Content created by you or
 stored in your user account, for evaluating the effectiveness of existing product features, planning
 new features, or measuring size and characteristics about specific end-users, or devices with a data
 broker, for commercial, marketing or any similar purpose. 
-unless specifically permitted by you, your
-use of the mobile application does not grant us the
-license to use analytics tools, advertising networks, third-party sdks, or other external vendors
-whose code we have added to the uni mobile application, to collect information such as user id,
-device id, or profiles with third-party data for the purpose of accessing necessary data to
+Unless specifically permitted by you, your
+use of the Mobile Application does not grant us the
+license to use analytics tools, advertising networks, third-party SDK's, or other external vendors
```

Comment:
When talking about multiple Software Development Kits, go with "SDKs" (instead of "SDK's").